File: module07/module39/module2/module2/src/main/java/com/android/example/module07_module39_module2_module2/ClassAAE.java
package com.android.example.module07_module39_module2_module2;

public class ClassAAE {
	private 	com.android.example.module07_module39_module2_module1.ClassAAH instance_var_1_0 = new com.android.example.module07_module39_module2_module1.ClassAAH();
	private 	com.android.example.module07_module39_module2_module1.ClassAAB instance_var_1_1 = new com.android.example.module07_module39_module2_module1.ClassAAB();
	private 	com.android.example.module07_module39_module2_module5.ClassAAE instance_var_1_2 = new com.android.example.module07_module39_module2_module5.ClassAAE();
	private 	com.android.example.module07_module39_module2_module5.ClassAAB instance_var_1_3 = new com.android.example.module07_module39_module2_module5.ClassAAB();
	private 	com.android.example.module07_module39_module2_module5.ClassAAJ instance_var_1_4 = new com.android.example.module07_module39_module2_module5.ClassAAJ();

	public void method0(
		com.android.example.module07_module39_module2_module5.ClassAAE param0,
		com.android.example.module07_module39_module2_module1.ClassAAE param1,
		com.android.example.module07_module39_module2_module1.ClassAAA param2,
		com.android.example.module07_module39_module2_module1.ClassAAB param3) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.cancel();

		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_5 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_5.isDisposed();

		for (int iAb = 0; iAb < 0; iAb++) {
			io.reactivex.internal.disposables.ListCompositeDisposable local_var_3_0 = new io.reactivex.internal.disposables.ListCompositeDisposable();
			local_var_3_0.clear();

			io.reactivex.schedulers.TestScheduler local_var_3_1 = new io.reactivex.schedulers.TestScheduler();
			local_var_3_1.createWorker();

			local_var_3_0.isDisposed();

			local_var_3_0.dispose();

		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_6 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_6.requestMore(-915821865l);

	}

	public void method1(
		com.android.example.module07_module39_module2_module5.ClassAAG param0,
		com.android.example.module07_module39_module2_module1.ClassAAA param1,
		com.android.example.module07_module39_module2_module1.ClassAAA param2,
		com.android.example.module07_module39_module2_module1.ClassAAJ param3) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_4 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_4.assertNotSubscribed();

		param0.method0(new io.reactivex.subscribers.TestSubscriber());

		param0.method2(new io.reactivex.subscribers.TestSubscriber());

		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_5 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_5.clear();

	}

	public void method2(
		com.android.example.module07_module39_module2_module1.ClassAAE param0) throws Throwable {
	}

	public void method3(
		com.android.example.module07_module39_module2_module5.ClassAAB param0) throws Throwable {
	}

	public void method4(
		com.android.example.module07_module39_module2_module1.ClassAAF param0,
		com.android.example.module07_module39_module2_module5.ClassAAF param1,
		com.android.example.module07_module39_module2_module5.ClassAAH param2) throws Throwable {
		param0.method0(new com.android.example.module07_module39_module2_module5.ClassAAI(), new com.android.example.module07_module39_module2_module5.ClassAAH());

	}
}

File: module06/module076/module4/src/main/java/com/android/example/module06_module076_module4/ClassAAE.kt
package com.android.example.module06_module076_module4

@Suppress("UNUSED_PARAMETER")
class ClassAAE {
	private 	val instance_var_1_0: com.android.example.module06_module076_module5.ClassAAF = com.android.example.module06_module076_module5.ClassAAF()
	private 	val instance_var_1_1: com.android.example.module06_module076_module1.ClassAAG = com.android.example.module06_module076_module1.ClassAAG()
	private 	val instance_var_1_2: com.android.example.module06_module076_module5.ClassAAH = com.android.example.module06_module076_module5.ClassAAH()
	private 	val instance_var_1_3: com.android.example.module06_module076_module5.ClassAAG = com.android.example.module06_module076_module5.ClassAAG()
	private 	val instance_var_1_4: com.android.example.module06_module076_module1.ClassAAD = com.android.example.module06_module076_module1.ClassAAD()

	fun method0(
		param0: com.android.example.module06_module076_module5.ClassAAA
	) {
		param0.method2("SomeString", "SomeString", "SomeString")

		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.assertNotSubscribed()

	}

	fun method1(
		param0: com.android.example.module06_module076_module1.ClassAAG,
		param1: com.android.example.module06_module076_module1.ClassAAH,
		param2: com.android.example.module06_module076_module5.ClassAAJ,
		param3: com.android.example.module06_module076_module1.ClassAAI
	) {
		if (kotlin.Any().equals(kotlin.Any())) {
			val local_var_3_0: io.reactivex.internal.schedulers.SingleScheduler = io.reactivex.internal.schedulers.SingleScheduler()
			local_var_3_0.createWorker()

		}
	}

	fun method2(
		param0: com.android.example.module06_module076_module5.ClassAAF,
		param1: com.android.example.module06_module076_module1.ClassAAA,
		param2: com.android.example.module06_module076_module5.ClassAAG
	) {
	}

	fun method3(
		param0: com.android.example.module06_module076_module5.ClassAAH,
		param1: com.android.example.module06_module076_module1.ClassAAE,
		param2: com.android.example.module06_module076_module5.ClassAAG,
		param3: com.android.example.module06_module076_module5.ClassAAD
	) {
		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.onNext(kotlin.Any())

		val local_var_2_5: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_5.dispose()

		param0.method1("SomeString", "SomeString", "SomeString")

		param0.method1("SomeString", "SomeString", "SomeString")

	}

	fun method4(
		param0: com.android.example.module06_module076_module1.ClassAAB
	) {
		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.onError(kotlin.Throwable())

		val local_var_2_2: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_2.dispose()

		val local_var_2_3: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_3.dispose()

		val local_var_2_4: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_4.isDisposed()

	}
}
